.ui/pgHomeSlider.pgx

```diff
@@ -60,6 +60,7 @@
 					"flexGrow": 1
 				},
 				"height": null,
+				"imageFillType": "ASPECTFIT",
```

Comment:
we shouldn't be using user props inside page

Reply:
done.